app/healthcheck.js

```diff
@@ -29,7 +29,7 @@ function createPromisesList(services) {
 
 function getServiceHealthUrl(url) {
```

Comment:
Perhaps we could remove the duplication from this method by using a ternary for the port, something like this?
```
function getServiceHealthUrl(url) {
    const port = url.port ? `:${url.port}` : '';
    return `${url.protocol}//${url.hostname}${port}'/health'`;
}
```